src/EA.Weee.RequestHandlers.Tests.Unit/Admin/AatfReports/GetAatfObligatedDataHandlerTests.cs

```diff
@@ -97,6 +99,7 @@ public async Task GetAatfObligatedDataCsvHandler_ReturnsFileName()
 
             // Assert
             Assert.Contains("2019_Q1", data.FileName);
+            Assert.Contains(datetime, data.FileName);
```

Comment:
Change the assert to not contains but the actual file name as we should know what it is... Can use the SystemTIme freeze